UI/platform-osx.mm

```diff
@@ -73,10 +73,25 @@ string GetDefaultVideoSavePath()
 {
 	NSFileManager *fm = [NSFileManager defaultManager];
 	NSURL *url = [fm URLForDirectory:NSMoviesDirectory
-				inDomain:NSUserDomainMask
-		       appropriateForURL:nil
-				  create:true
-				   error:nil];
+			inDomain:NSUserDomainMask
```

Comment:
Looks like you included changes which don't belong in this commit (and PR)

Reply:
this is styling (whitespace) change requested by Jim in the previous set of comments

Reply:
It's not, he requested styling to change on the lines you added in that commit, not existing lines

Reply:
when you see a broken window, you fix it?

Reply:
By breaking git history? Nope